app/templates/attendee-app.hbs

```diff
@@ -84,7 +84,7 @@
         </div>
       </div>
       <div class="eight wide column">
-        <img class="ui fluid image" src="/images/android-img.jpg" alt="Android Pic">
+        <img class="ui medium image" src="/images/android-img.jpg" alt="Android Pic">
```

Comment:
Can you update it with ~~medium~~ large and check if there is any empty right space as present in medium size?
If not, then use the large and its LGTM

Reply:
@vishakhanihore ,
over where the [issue ](https://github.com/fossasia/open-event-frontend/issues/3529#issuecomment-543315688)is open, you can see a screenshot I added for how the page looks when using the large size. Please refer to it there and suggest what you think is more appropriate.

Reply:
@vishakhanihore , @kushthedude , anyone want to comment?